atomic_reactor/plugin.py

```diff
@@ -408,8 +408,8 @@ def __init__(self, dt, workflow, plugin_conf, *args, **kwargs):
         logger.info("initializing runner of build-step plugin")
         self.plugins_results = workflow.buildstep_result
 
-        if not plugin_conf:
-            # if none buildstep plugins specified, fallback to docker api plugin
+        if plugin_conf is None:
```

Comment:
We want this because we may want to explicitly not have any buildstep_plugin executed?